CONTRIBUTING.md

```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``. 
 
+Let's say you're announcing a cool new project you've recently released. Your file would be named something like ``2023-04-22-cool-project-v1-release.md``.
```

Comment:
We can also have them put the blog date in the metadata, but I figured having it in the file name would be a good way to organize the blog files themselves and Docusaurus can strip out and use the dates if presented this way in the file name, to my knowledge.

Reply:
That's a good idea.